repro.in

```diff
@@ -528,14 +531,20 @@ General Options:
  -d                           Run diffoscope if packages are not reproducible
  -f                           Use the local PKGBUILD for building
  -n                           Run makepkg with --nocheck
+ -v                           Print version information
  -o <path>                    Set the output directory (default: ./build)
 __END__
 }
 
+function print_version() {
+    echo "repro ${VERSION}"
+}
+
 function parse_args() {
-    while getopts :hdnf arg; do
+    while getopts :hdnfv arg; do
         case $arg in
             h) print_help; exit 0;;
+            v) print_version; exit 0;;
```

Comment:
I'd suggest `-V`, it's more common for `--version` and `-v` is usually `--verbose`.

Reply:
Yep, this is nicer. Should we print this when building packages for the log as well?